### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by robinedman**
Fixes 2 noting bugs.

## Bug 1: Can't expand note
**To replicate you can for example do:**

1. Make some text bold.
2. Create a note that starts before the bold text and ends after it.
3. Collapse it.
4. Try to expand it.

## Bug 2: Text is selected when expanding a note

**To replicate:**
Try expanding a note. Either the whole or part of the note will be highlighted.

## Bug 3: https://github.com/guardian/scribe-plugin-noting/issues/133

### CODE REVIEW

**Comment by hmgibson23** [comment]
Nice fix :+1:

**Comment by robinedman** [comment]
I added the third bug fix to this PR

File: src/actions/noting/toggle-note-classes.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = function toggleNoteClasses(notes, className) {
     action = toggleClass;
   } else {
     //if we have more than one note then we want them all to share state
-    var state = collapseState.get();
+    var state = notes.every(noteSegment => hasClass(noteSegment.vNode, className));
```
Comment (cutandpastey) [inline src/actions/noting/toggle-note-classes.js]: Does this make `collapseState` redundant?

Comment (robinedman) [inline src/actions/noting/toggle-note-classes.js]: No `collapseState` is used when you collapse/expand _all_ notes. As far as I could tell, it was incorrectly used here.

Comment (cutandpastey) [inline src/actions/noting/toggle-note-classes.js]: @robinedman good shout.

**Comment by cutandpastey** [comment]
:+1:

**Comment by robinedman** [comment]
Cheers! And tests pass so I'm merging this.

**Pull request merged**